core/src/test/java/org/apache/iceberg/jdbc/TestJdbcUtil.java

```diff
@@ -18,6 +18,22 @@
  */
 package org.apache.iceberg.jdbc;
 
+import static org.apache.iceberg.jdbc.JdbcUtil.createCatalogTableSql;
+import static org.apache.iceberg.jdbc.JdbcUtil.createCatalogViewSql;
+import static org.apache.iceberg.jdbc.JdbcUtil.dropTableSql;
+import static org.apache.iceberg.jdbc.JdbcUtil.dropViewSql;
+import static org.apache.iceberg.jdbc.JdbcUtil.filterAndRemovePrefix;
+import static org.apache.iceberg.jdbc.JdbcUtil.getTableSql;
+import static org.apache.iceberg.jdbc.JdbcUtil.getViewSql;
+import static org.apache.iceberg.jdbc.JdbcUtil.listTableNamespacesSql;
+import static org.apache.iceberg.jdbc.JdbcUtil.listTablesSql;
+import static org.apache.iceberg.jdbc.JdbcUtil.listViewNamespacesSql;
+import static org.apache.iceberg.jdbc.JdbcUtil.listViewsSql;
+import static org.apache.iceberg.jdbc.JdbcUtil.renameTableSql;
+import static org.apache.iceberg.jdbc.JdbcUtil.renameViewSql;
+import static org.apache.iceberg.jdbc.JdbcUtil.tableNamespaceSql;
+import static org.apache.iceberg.jdbc.JdbcUtil.viewNamespaceSql;
```

Comment:
The project prefers not to use static imports so that you can clearly see where methods are defined.

Reply:
I agree. It was a request from @nastra (maybe I misunderstood). I will revert.

Reply:
sorry if it wasn't clear, but I actually meant to only statically import `assertThat()` from `Assertions` in my [comment](https://github.com/apache/iceberg/pull/9487#discussion_r1458873570) :)

Reply:
Ah ok my bad 😃